src/impl_error_chain_kind.rs

```diff
@@ -85,8 +85,7 @@ macro_rules! impl_error_chain_kind {
         queue [ #[$qmeta:meta] $( $tail:tt )*]
     ) => {
         impl_error_chain_kind!(SORT [$( $def )*]
-            enum [$( $(#[$emeta])* => $eitem $(( $($etyp),* ))* )*
```

Comment:
```
error: unknown macro variable `emeta`
  --> src/impl_error_chain_kind.rs:88:26
   |
88 |             enum [$( $(#[$emeta])* => $eitem $(( $($etyp),* ))* )*
   |                          ^^^^^^
```
```
error: unknown macro variable `eitem`
  --> src/impl_error_chain_kind.rs:88:39
   |
88 |             enum [$( $(#[$emeta])* => $eitem $(( $($etyp),* ))* )*
   |                                       ^^^^^^
```
```
error: unknown macro variable `etyp`
  --> src/impl_error_chain_kind.rs:88:52
   |
88 |             enum [$( $(#[$emeta])* => $eitem $(( $($etyp),* ))* )*
   |                                                    ^^^^^
```

Someone familiar with the code should be able to decide how to fix this error:
- Either remove the whole rule because it is never reached (most probable actually).
- Or adapt the right-hand side of the rule such that it does not reference unbound meta-variables (that's what I tried to do).
- Or adapt the left-hand side of the rule such that it binds `$emeta`, `$eitem`, and `$etyp`.